### PR TITLE
Update quests.md

### PR DESCRIPTION
**Pull request by frantz2501**
Proposal entry for Quests page

### CODE REVIEW

File: docs/source/entities/quests.md [hunk 1]
```diff
@@ -1 +1,20 @@
-# Quests
\ No newline at end of file
+# Quests
+
+Quests are a place to regroup all elements for a part of scenario or a scene.
+
+## Features
+Quests specific features are:
+* can be nested with "parent quest" field
+* a "date" field for reporting which will be displayed in side bar below pinned attributes
```
Comment (ilestis) [inline docs/source/entities/quests.md:8]: You can replace "side bar" with `[profile sidebar](/features/profile-sidebar)` to keep it consistent with the rest of the docs

File: docs/source/entities/quests.md [hunk 2]
```diff
@@ -1 +1,20 @@
...
+* a "completed" field to keep track of ... completed quest in list view
+* calendar date entry: when filled up, it will automatically populate the calendar with reminder
```
Comment (ilestis) [inline docs/source/entities/quests.md:10]: Please replace `reminder` with `a [reminder](/features/reminders)`

File: docs/source/entities/quests.md [hunk 3]
```diff
@@ -1 +1,20 @@
...
+* an elements subpage
+
+## The elements subpage
+The Elements subpage allow you to collect all elements of a quest, being characters, locations, events, objects, notes, journals and so on.
+
+![Screenshot from 2023-04-18 09-43-21](https://user-images.githubusercontent.com/16939798/232707341-993dc66d-9cb0-479d-80ee-0b1dee52ad37.png)
```
Comment (ilestis) [inline docs/source/entities/quests.md:16]: Images should be hosted directly in the git for history and avoiding people replacing external images with porn or similar content :) You you add them to `docs/source/entities/img/` and prefix the files with `quests-xyz.png` and include them with `Explaining the image`

File: docs/source/entities/quests.md [hunk 4]
```diff
@@ -1 +1,20 @@
...
+
+This way, you have at hand all usefull elements to keep track of your quest as well as subquests.
+Each reference has a "Role" field which all you to group and sorts elements in the Elements subpage.
+![quests](https://user-images.githubusercontent.com/16939798/232708216-f5c39959-dd0d-4583-873c-83189ae200dc.png)
```
Comment (ilestis) [inline docs/source/entities/quests.md:20]: Same thing, please host the images in the git and not external to the docs

**Review by ilestis — CHANGES_REQUESTED** [review]
Thanks for the PR! Biggest issue is images are hosted externally meaning you could change them with images for porn, or worse like crypto. Please add the screenshots directly to the git and follow the convention used throughout the docs :)

**Comment by frantz2501** [comment]
done as per your instructions !

File: docs/source/entities/quests.md (hunk 4 above)
Comment (frantz2501) [reply docs/source/entities/quests.md]: done

**Pull request merged**